src/pages/Root/Root.js

```diff
@@ -83,11 +82,10 @@ const Root = ({
 			const synths = snxJSConnector.snxJS.contractSettings.synths.filter(synth => synth.asset);
 			setAvailableSynths(synths);
 			fetchAndSetExchangeData(synths);
-			fetchAndSetWalletBalances(synths);
 			clearInterval(intervalId);
 			intervalId = setInterval(() => {
 				fetchAndSetExchangeData(synths);
-				fetchAndSetWalletBalances(synths);
+				fetchWalletBalances();
```

Comment:
`fetchWalletBalances` no longer needs to get `synths` since it will read the latest ones from the store